src/bp/ui-studio/src/web/views/OneFlow/diagram/VariableTypesForm/index.tsx

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios'
+import sdk from 'botpress/sdk'
+import { RightSidebar } from 'botpress/shared'
+import _ from 'lodash'
+import React, { FC } from 'react'
+import { connect } from 'react-redux'
+import { deleteEntity, refreshEntities, setActiveFormItem } from '~/actions'
+import { ActiveFormItem } from '~/reducers/flows'
+
+import style from '../PromptForm/style.scss'
+
+import EnumForm from './EnumForm'
+import PatternForm from './PatternForm'
+
+interface Props {
+  customKey: string
+  contentLang: string
+  formData: sdk.NLU.EntityDefinition
+  close: () => void
+  deleteEntity: (entityId: string) => void
+  refreshEntities: () => void
+  setActiveFormItem: (data: ActiveFormItem) => void
+}
+
+export const getEntityId = (entityName: string) =>
+  entityName
+    .trim()
+    .toLowerCase()
+    .replace(/[\t\s]/g, '-')
+
+const VariableForm: FC<Props> = props => {
+  const updateEntity = _.debounce(async (originalId: string, entity) => {
+    if (entity.name) {
+      await axios.post(`${window.BOT_API_PATH}/nlu/entities/${originalId}`, entity)
+      props.refreshEntities()
+    }
+  }, 100)
+
+  const deleteEntity = async (entityId: string) => {
+    props.deleteEntity(entityId)
+    props.refreshEntities()
+    props.close()
+  }
+
+  const updateFormItem = (data: sdk.NLU.EntityDefinition) => {
+    props.setActiveFormItem({ type: 'entity', data })
```

Comment:
as a general comment can we rename form type 'entity' for variableType ? as the concept of entity goes away (we'll keep this only in nlu and hidden in the form which interacts with nlu backend)

We don't need to rename every occurrence to entity but at least for the formType